Marlin/src/lcd/extui/mks_ui/wifi_module.cpp

```diff
@@ -863,11 +882,12 @@ static void wifi_gcode_exec(uint8_t *cmd_line) {
           file_writer.fileTransfer = 0;
           if (uiCfg.print_state == IDLE) {
             int index = 0;
+            bool with_longnames = strchr((char *)tmpStr, 'L') != NULL;
 
             if (tmpStr == 0) {
```

Comment:
NOTE: If `tmpStr` is a `nullptr` then `with_longnames` is definitely `false` so there is no need to pass it in `get_file_list` below.